ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -486,9 +754,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-boot-categories-project
+- as: nutanix-ipi-boot-categories-project-rt-f28
```

Comment:
If we add this new job, we can remove the `nutanix-ipi-boot-categories-project` one.

Reply:
Removed `nutanix-ipi-boot-categories-project`